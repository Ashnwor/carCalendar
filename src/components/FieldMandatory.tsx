import React from 'react';
import { HelperText } from 'react-native-paper';

const FieldMandatory = () => {
	return <HelperText type="error">*Bu alan zorunlu</HelperText>;
};

export default FieldMandatory;
