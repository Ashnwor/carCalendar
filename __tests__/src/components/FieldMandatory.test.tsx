import React from 'react';
import { render } from 'react-native-testing-library';

import FieldMandatory from '../../../src/components/FieldMandatory';

describe('<FieldMandatory />', () => {
	it('renders without crashing', () => {
		const component = render(<FieldMandatory />);
		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
