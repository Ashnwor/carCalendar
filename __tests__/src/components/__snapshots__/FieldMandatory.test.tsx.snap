// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FieldMandatory /> renders without crashing 1`] = `
<Text
  onLayout={[Function]}
  style={
    Object {
      "color": "#B00020",
      "fontFamily": "System",
      "fontSize": 12,
      "fontWeight": "400",
      "opacity": 1,
      "paddingHorizontal": 12,
      "paddingVertical": 4,
      "textAlign": "left",
      "transform": Array [
        Object {
          "translateY": -0,
        },
      ],
      "writingDirection": "ltr",
    }
  }
>
  *Bu alan zorunlu
</Text>
`;
